containers/kubernetes/how-to/use-scratch-storage-h100.mdx

```diff
@@ -14,26 +14,26 @@ categories:
   - kubernetes
 ---
 
-[Scratch storage](/compute/gpu/concepts/#scratch-storage) is ephemeral storage allocated to H100 GPU Instances (up to 6 TB) and crucial for handling temporary data in high-performance computing environments, especially when working with GPUs like the H100. Kubernetes, a powerful container orchestration platform, can be configured to make optimal use of scratch storage for improved performance.
+[Scratch storage](/compute/gpu/concepts/#scratch-storage) is ephemeral storage allocated to H100 and L40S GPU Instances (up to 6 TB) and crucial for handling temporary data in high-performance computing environments, especially when working with GPUs like the H100. Kubernetes, a powerful container orchestration platform, can be configured to make optimal use of scratch storage for improved performance.
 
 Its utility spans multiple use cases, including but not limited to:
 
-- **Large dataset storage for training:** H100 scratch storage provides a local, high-performance storage option for storing large datasets required for training machine learning models. This can significantly enhance the efficiency of training workflows by reducing data transfer times.
+- **Large dataset storage for training:** H100 and L40S scratch storage provides a local, high-performance storage option for storing large datasets required for training machine learning models. This can significantly enhance the efficiency of training workflows by reducing data transfer times.
 - **Checkpointing during training:** The scratch volume allows for the seamless saving of checkpoints during model training. This is important for resuming training processes in case of interruptions or for exploring different model iterations without starting from the beginning.
-- **Temporary storage for preprocessing:** For data preprocessing tasks that demand high I/O throughput and low latency, H100 scratch storage offers a temporary space on the host node's filesystem. This can be advantageous for quick and efficient preprocessing steps in complex data pipelines.
-- **Experimentation and output evaluation:** Experimenting with different configurations and hyperparameters often involves iterative processes. H100 scratch storage makes quick storage and retrieval of intermediary results easy, enabling efficient experimentation and evaluation of model outputs.
+- **Temporary storage for preprocessing:** For data preprocessing tasks that demand high I/O throughput and low latency, H100 and L40S scratch storage offers a temporary space on the host node's filesystem. This can be advantageous for quick and efficient preprocessing steps in complex data pipelines.
+- **Experimentation and output evaluation:** Experimenting with different configurations and hyperparameters often involves iterative processes. H100 and L40S scratch storage makes quick storage and retrieval of intermediary results easy, enabling efficient experimentation and evaluation of model outputs.
```

Comment:
I'm wondering if it should be singular here: `H100 and L40S scratch storage **make** quick storage and retrieval...`
@RoRoJ